src/main/java/org/jabref/gui/preftabs/PreviewPrefsTab.java

```diff
@@ -126,8 +131,19 @@ private void setupLogic() {
                 DefaultTaskExecutor.runInJavaFXThread(() -> {
 
                     PreviewPanel testPane = new PreviewPanel(null, null, Globals.getKeyPrefs(), Globals.prefs.getPreviewPreferences(), dialogService);
-                    testPane.setFixedLayout(layout.getText());
-                    testPane.setEntry(TestEntry.getTestEntry());
+                    if (chosen.isSelectionEmpty()) {
+                        testPane.setFixedLayout(layout.getText());
+                        testPane.setEntry(TestEntry.getTestEntry());
+                    }
+                    else {
+                        int indexStyle = chosen.getSelectedIndex();
+                        PreviewPreferences p = Globals.prefs.getPreviewPreferences();
```

Comment:
Please don't use abbreviations as this makes the code harder to understand: `p -> preferences`.

Reply:
Sorry about that, I changed it to preferences, and added a changelog entry as well. Thank you.